rita_common/src/dashboard/wallet.rs

```diff
@@ -54,8 +58,8 @@ fn withdraw_handler(address: Address, amount: Option<Uint256>) -> HttpResponse {
         };
 
     let tx_cost = gas_price * tx_gas;
-    if amount.clone() + tx_cost.clone() >= balance {
-        amount = balance - tx_cost;
+    if amount.clone() + tx_cost.clone() >= balance.clone().unwrap_or_else(|| 0u64.into()) {
+        amount = balance.unwrap_or_else(|| 0u64.into()) - tx_cost;
```

Comment:
this should handle the none case explicitly so it can send a proper 'not ready' error message rather than trying to create a transaction with a value of zero, which can actually execute and cost gas money.